ci/docker/runtime_functions.sh

```diff
@@ -489,9 +464,8 @@ build_ubuntu_amalgamation_min() {
 build_ubuntu_gpu_cmake_mkldnn() {
     set -ex
     cd /work/build
+    install_ccache_wrappers
```

Comment:
There should not be any need for wrapper for cmake based builds.

Reply:
The idea here is that we rather intercept the backend calls to the compilers instead of defining them through the frontend (aka the Cmake parameters). This allows us to be flexible with whatever compilers we define without having to think about ccache because it's handled automatically. Does that make sense?

Reply:
As much as I know cmake will not like this and different problems can appear suddenly. Using cmake parameters is the recommended approach.

Reply:
I know and I only chose this approach after playing around a lot. The problem is that the underlying submodules we have don't respect that choice.